pkg/clients/coredata/event.go

```diff
@@ -469,3 +470,38 @@ func (e *EventRestClient) DeleteOld(age int) error {
 
 	return nil
 }
+
+// Mark event as pushed
+func (e *EventRestClient) MarkPushed(id string) error {
```

Comment:
Please provide a unit test in event_test.go for this new method.

Reply:
Will do

Reply:
@tsconn23 I have pushed couple commits adding a test case and change make code in goroutine in line. Let me know if you have further concerns except for @drasko 's below.